seatunnel-connectors-v2/connector-hbase/src/main/java/org/apache/seatunnel/connectors/seatunnel/hbase/format/HBaseDeserializationFormat.java

```diff
@@ -60,16 +60,15 @@ private Object deserializeValue(SeaTunnelDataType<?> typeInfo, byte[] cell) {
             case TINYINT:
             case SMALLINT:
             case INT:
-                return Bytes.toInt(cell);
+                return Integer.valueOf(Bytes.toString(cell));
             case BOOLEAN:
-                return Bytes.toBoolean(cell);
+                return Boolean.valueOf(Bytes.toString(cell));
             case BIGINT:
-                return Bytes.toLong(cell);
+                return Long.valueOf(Bytes.toString(cell));
             case FLOAT:
             case DECIMAL:
-                return Bytes.toFloat(cell);
             case DOUBLE:
-                return Bytes.toDouble(cell);
+                return Double.valueOf(Bytes.toString(cell));
```

Comment:
**Scalability Issue**: The change from direct byte conversion methods (e.g., Bytes.toInt) to parsing strings (Integer.valueOf(Bytes.toString(cell))) introduces unnecessary overhead and potential performance degradation, especially under high load scenarios where efficient data processing is critical for scalability. <br> **Fix**: Revert to using direct byte conversion methods which are more efficient in terms of both CPU and memory usage, as they avoid the overhead of string creation and parsing. <br> **Code Suggestion**: 
 ```
 -                return Integer.valueOf(Bytes.toString(cell));
 +                return Bytes.toInt(cell);
 -                return Boolean.valueOf(Bytes.toString(cell));
 +                return Bytes.toBoolean(cell);
 -                return Long.valueOf(Bytes.toString(cell));
 +                return Bytes.toLong(cell);
 -                return Double.valueOf(Bytes.toString(cell));
 +                return Bytes.toDouble(cell);
 ```